website/docs/quick-start/installation/modal/app.py

```diff
@@ -18,7 +23,22 @@ def download_model():
             "download",
             "--model",
             MODEL_ID,
-        ]
+        ],
+        env=TABBY_ENV,
+    )
+
+
+def download_chat_model():
+    import subprocess
+
+    subprocess.run(
+        [
+            "/opt/tabby/bin/tabby-cpu",
```

Comment:
we can use `/bin/tabby` directly now - tabby-cpu is mostly for backward compatibility.

Reply:
Roger that. A mocal GPU machine is used in our example and it is true that /bin/tabby should be used instead of bin/tabby-cpu.

Reply:
I use the TABBY_BIN constant to refer to tabby binaries.